packages/npm-bcrypt/wrapper.js

```diff
@@ -27,7 +27,9 @@ try {
     "",
     "  meteor npm install --save bcrypt",
     "",
-    "in the root directory of your application."
+    "in the root directory of your application. You may also need to",
+    "install build tools to compile bcrypt. For more information see:,
```

Comment:
```suggestion
    "install build tools to compile bcrypt. For more information see:",
```
That Travis test failure was unfortunately related to this PR.